main/guides/zoe/index.md

```diff
@@ -1,24 +1,38 @@
 # Zoe Overview
 
-## What is Zoe?
+The Zoe service and smart contract API support credibly trading assets with reduced risk.
 
-Zoe is the heart of Agoric's smart contract framework. Use Zoe to:
-
-- **Run your code on-chain**
-- **Mint new digital assets**
-- **Credibly trade assets**
-
-## Why Use Zoe?
-
-- **Zoe is Safer for Users:** A Zoe offer guarantees that you get either
+- **Zoe is Safer for Users:** Zoe guarantees that when you make an offer, you get either
   what you wanted or a full refund of the assets you put in,
   even if the contract is buggy or malicious.
 
-- **Zoe is Safer for Developers** For a given offer, if you
+- **Zoe is Safer for Developers**: For a given offer, if you
   make a mistake with the amount of assets you take or give,
   Zoe guarantees that your users will either get what they say
   they wanted or get a refund.
 
+## High Level Trading Flow
+
+Trading with a contract using Zoe typically goes through these steps:
+
+<img src="./assets/offer-safety-flow.svg" alt="offer safety flow with contracts, zoe, and parties"
+ style="border: 2px solid" />
+
+1. Parties call [E(zoe).offer(invitation, proposal, assets)](/reference/zoe-api/zoe#e-zoe-offer-invitation-proposal-paymentpkeywordrecord-offerargs). Zoe escrows the assets.
+2. Zoe relays the proposal to the contract identified in the invitation.
+3. The contract handles proposals using its custom business logic.
+4. The contract instructs Zoe to [reallocate](/reference/zoe-api/zoe-contract-facet#zcf-reallocate-seats) assets among the parties.
+5. The contract completes (aka [exits](/reference/zoe-api/zcfseat#azcfseat-exit-completion)) the offers.
+6. Zoe pays out assets to the parties.
+
+Note that in this flow, _assets are not sent to the contract_; only information about them.
```

Comment:
I consider [my blog post](https://agoric.com/blog/technology/the-settlers-of-blockchain) to be a good explanation of this. This would be a fine place to link it.